ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Types.hs

```diff
@@ -613,6 +613,8 @@ data TracePeerSelection peeraddr =
      | TraceGovernorWakeup
      | TraceChurnWait          DiffTime
      | TraceChurnMode          ChurnMode
+     | TracePeerNess           Int (Double, Double, Double, Double, Double)
+                               Int (Double, Double, Double, Double, Double)
```

Comment:
Please add a haddock which explains the fields.

Reply:
I will remove the commit. It doesn't belong in this PR.